src/app.css

```diff
@@ -0,0 +1,232 @@
+:root {
+  --blue: #155da1;
+  --blue-light: #3e87cd;
+  --blue-lighter: #b4d8fa;
+  --blue-dark: #0e4e8a;
+  --blue-darker: #0d3f6e;
+  --gray-lighter: #ebeef2;
+  --gray-light: #8b9199;
+  --text-dark: #031425;
+
+  --header-width: min(1440px, 90vw);
+  --content-width: min(1200px, 85vw);
+  --long-form-width: min(900px, 80vw);
+
+  --animation: 0.2s ease;
```

Comment:
I intentially inlined global CSS in app.html - is there a reason you separated it out? I was under the impression that inlining global styles was a good idea to cut down on a network request, but I'm not an expert on this.

Reply:
I separated it out because I've seen this pattern a lot for SvelteKit, and it's how [the template](sveltekit.new) is setup. It does add a network request, but this can be avoided with [inlineStyleThreshold](https://kit.svelte.dev/docs/configuration#inlinestylethreshold). That said, the file is a bit large to set an inline threshold for in my opinion, so in the future I might try to shrink it by putting some of these styles into components and removing them from here, but for now I'll just revert this back to app.html. I'll still keep the inline style threshold at 1024 bytes for any small files currently.

The only downside to inlining css manually is that Vite's dev server doesn't apply HMR to edits. Also, I think it's a bit easier to read when separated into its own file.